src/frontend/js/widgets/Dashboard/index.spec.tsx

```diff
@@ -41,10 +42,11 @@ jest.mock('hooks/useIntersectionObserver', () => ({
 }));
 
 describe('<Dashboard />', () => {
-  const DashboardWithUser = ({ user }: { user: Nullable<User> }) => {
+  let user: UserEvent;
```

Comment:
I would prefer to declare `const user = userEvent.setup()` in each test suite where it is needed.